src/utils/index.js

```diff
@@ -57,25 +57,24 @@ export function log(message) {
 }
 
 // Refresh the redux store
-export function updateEvents() {
-    fetchBackend('/events', 'GET')
-        .then(response => {
+export async function updateEvents() {
```

Comment:
does this return a promise? Wondering if wrapping this inside a Promise.all is actually correct

Reply:
this confused me for the longest time LOL.. calling an async function without using `await` returns a promise

Reply:
And we want to return a promise right?

Reply:
yup.. `Promise.all` accepts an iterable of promises